src/Symfony/Component/HttpFoundation/Session/Attribute/NamespacedAttributeBag.php

```diff
@@ -43,7 +43,7 @@ public function __construct($storageKey = '_sf2_attributes', $namespaceCharacter
      */
     public function has($name)
     {
-        $attributes = $this->resolveAttributePath($name);
+        $attributes = &$this->resolveAttributePath($name);
```

Comment:
No pb for adding a comment.
You're right, there is no ref safe way for these functions...